src/status_im/ui/screens/wallet/request/subs.cljs

```diff
@@ -8,4 +8,4 @@
   (fn [[amount amount-error]]
     (and
       (nil? amount-error)
-      (not (nil? amount)) (not= amount ""))))
```

Comment:
Why are we removing this validation step?

Reply:
Amount is a bignumber now.